tests/unit/challenges/test_models.py

```diff
@@ -96,6 +97,29 @@ def test_get_end_date(self):
                          self.challenge.get_end_date())
 
 
+class DatasetSplitTestCase(BaseTestCase):
+
+    def setUp(self):
+        super(DatasetSplitTestCase, self).setUp()
```

Comment:
The idea behind making a temporary directory is to upload all files related to a model here. After everything is done this directory can simply be removed. Also mostly os manages the temp directory allocation and removal by themselves.

But if we see DatasetSplit model, there is no file field.  So I think you can remove this part from here. Same goes for LeaderBoard model.